### DISCUSSION TITLE
Well...

### DISCUSSION THREAD

**Discussion by icarusinthewings** [NovaoTech/solaris#36] [General]
https://polarisprogram.com/
:L

**Comment by spazestudios** [comment]
rip how about polaris.fakedomain polariscode.fakedomain

**Comment by udscminidisc** [comment]
it's pArKEd


​
________________________________
From: Judah B. ***@***.***>
Sent: Friday, February 18, 2022 5:19 PM
To: JTechnologies/Polaris ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [JTechnologies/Polaris] Well... (Discussion #36)


https://polarisprogram.com/
:L

—
Reply to this email directly, view it on GitHub<https://github.com/JTechnologies/Polaris/discussions/36>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AU7ZMLZ7H7IMMRTG67PYHQLU33AY3ANCNFSM5OZHKL5Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by udscminidisc** [comment]
> https://polarisprogram.com/ :L

HahaaAhhaaa

did We TeaM wIt NasA?
